packages/styled-components/components/container/index.js

```diff
@@ -17,6 +17,8 @@ const widths = {
  * @todo Is there a better way to handle the maxWidth and background image
  *       functionality? We want this functionality to be independent from
  *       themes.
+ * @todo If children is empty, should we render anything? Do we want a
```

Comment:
I would lean towards not rendering anything at all in this case. IMO it doesn't make sense to send empty markup to the DOM that won't be used in any meaningful way.

Reply:
Cool. I agree, implemented.